### DISCUSSION TITLE
guide-R/analyses/selection-modele-pas-a-pas

### DISCUSSION THREAD

**Discussion by giscus** [larmarange/guide-R#46] [Forum]
# guide-R/analyses/selection-modele-pas-a-pas

https://larmarange.github.io/guide-R/analyses/selection-modele-pas-a-pas.html

**Comment by WilliamZ98** [comment]
Bonjour monsieur.
Merci pour ce cours.
J'aimerai vous posez une question qui pourrait semblé évidente : est-ce que les méthodes de sélection ascendante et descendante aboutissent-elles toujours au même résultats conformément au cas que vous avez présenté ? Car cela pourrait conditionner notre choix entre les deux méthodes.
Merci !

**Reply by larmarange** [reply]
C'est une bonne question à laquelle je n'aurais pas de réponse certaine. Il me semble que pour les modèles simples cela aboutisse presque toujours au même résultat, mais je ne peux exclure des cas particuliers atypiques.

A l'usage, pour un modèle simple, la sélection descendante est la plus courante. Il s'agit de simplifier le modèle. Pour ma part, je recours à une sélection ascendante plutôt quand je cherche à explorer des interactions auxquelles je n'aurai pas pensé. Je pars alors d'un modèle sans interaction (éventuellement déjà simplifié par une sélection ascendante) que je cherche à compléter avec des interactions pertinentes (voir un exemple ici : https://larmarange.github.io/guide-R/analyses/interactions.html#identifier-les-interactions-pertinentes ). Comme le modèle maximal possible contient beaucoup interactions, il n'arriverait pas facilement à converger et donc une approche descendante ne serait pas appropriée. Ici, une approche ascendante permet de compléter le modèle.

**Reply by WilliamZ98** [reply]
Merci monsieur pour votre réponse.
